src/Nox.Generator/Presentation/Api/OData/EntityControllerGenerator.cs

```diff
@@ -112,10 +112,10 @@ public void Generate(SourceProductionContext context, NoxSolutionCodeGeneratorSt
             if (entity.Persistence is null ||
                 entity.Persistence.Read.IsEnabled)
```

Comment:
@ricardo Is there any reason for generating Owned Relationships only if the Entity is readable?